### PR TITLE
Radau quadrature for SDC

### PR DESCRIPTION
**Pull request by zingale**
## PR summary

This implements Radau quadrature for the SDC integration.  Addresses #664 

## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite
- [ ] all functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

**Comment by zingale** [comment]
blocked by #654

File: CHANGES [hunk 1]
```diff
@@ -1,7 +1,39 @@
+<<<<<<< HEAD
+=======
```
Comment (maxpkatz) [inline CHANGES]: Looks like merge conflicts that need to be fixed

File: Source/driver/Castro_advance_sdc.cpp [hunk 2]
```diff
@@ -207,6 +207,9 @@ Castro::do_advance_sdc (Real time,
     // this if we are on the final node (since there is nothing to
     // update to
     if (m < SDC_NODES-1) {
+
+      std::cout << "... doing the SDC update, iteration = " << sdc_iteration << " from node " << m << " to " << m+1 << std::endl;
```
Comment (maxpkatz) [inline Source/driver/Castro_advance_sdc.cpp]: Use amrex::Print() to avoid getting spammed for multiple ranks

File: Source/driver/sdc_util.F90 [hunk 3]
```diff
@@ -81,6 +83,13 @@ subroutine sdc_solve(dt_m, U_old, U_new, C, sdc_iteration)
 
        ! failing?
        if (ierr /= NEWTON_SUCCESS) then
+          print *, "Newton convergence failure"
```
Comment (maxpkatz) [inline Source/driver/sdc_util.F90]: Doesn't look like U_orig is being filled here

Comment (zingale) [reply Source/driver/sdc_util.F90]: fixed

**Pull request merged**